pandas/tests/io/json/test_normalize.py

```diff
@@ -242,11 +242,13 @@ def test_missing_field(self, author_missing_data):
         # unnest records where only the first record has a None value
```

Comment:
Can delete this comment (tbh I don't really understand what it means even with the current behavior)